spec/importer_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'spec_helper'
+require 'ronin/masscan/importer'
+require 'masscan/output_file'
+require 'ronin/db'
+
+RSpec.describe Ronin::Masscan::Importer do
+  let(:masscan_path) { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures', 'masscan.json')) }
+  let(:output_file)  { Masscan::OutputFile.new(masscan_path) }
+
+  before(:all) do
+    Ronin::DB.connect('sqlite3::memory')
+  end
+
+  after(:all) do
+    Ronin::DB::IPAddress.destroy_all
+    # Ronin::DB::Port.destroy_all
```

Comment:
I commented it because it raises `ActiveRecord::InvalidForeignKey:`

Reply:
Maybe try changing the order to avoid any destroy constraints.

```ruby
Ronin::DB::OpenPort.destroy_all
Ronin::DB::IPAddress.destroy_all
Ronin::DB::Port.destroy_all
```

At least that works in the `ronin-db-activerecord/spec/open_port_spec.rb` specs.